### PR TITLE
use standard csv.reader for loading csv file, fix bug of quotations in csv

### PR DESCRIPTION
**Pull request by ilius**

### CODE REVIEW

**Comment by pettarin** [comment]
Hi, apologies for replying so late, but I rely on email notifications (not the Web thing) to monitor GitHub events, and this one got buried among other stuff.

You are right on the fact that the code should use the `csv` module, instead of doing "manual parsing". CSV is supported mainly for manual/debugging inspection, rather for being a meaningful I/O format.

However, judging from the code alone, your changes seem to remove some functionalities, like the possibility for the user to specify the input file encoding or the field and line separators. Plus, the adoption of the "excel" dialect seems a bit arbitrary to me.

To summarize: I am not going to merge this pull. As I explained elsewhere, I cannot spend more time working on Penelope, including managing bugs and pull requests. ASAP, I will add a notice of that to the `README` soon. I think the most convenient way of proceeding consists in you copying/adapting code from Penelope into `pyglossary`. To this end, I will add a link to `pyglossary` in the `README`.
